tests/flytekit/common/workflows/presto.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+
+from flytekit.sdk.tasks import inputs
+from flytekit.sdk.types import Types
+from flytekit.sdk.workflow import workflow_class, Input, Output
+from flytekit.common.tasks.presto_task import SdkPrestoTask
+
+schema = Types.Schema([("a", Types.String), ("b", Types.Integer)])
+
+presto_task = SdkPrestoTask(
+    query="SELECT * FROM hive.city.fact_airport_sessions WHERE ds = '{{ds}}' LIMIT 10",
+    output_schema=schema.schema_type,
+    routing_group="{{ routing_group }}",
+    catalog="hive",
+    schema="city",
+    task_inputs=inputs(ds=Types.String, routing_group=Types.String),
```

Comment:
Using a decorator like this seems a bit confusing to me. Perhaps it would be better just to pass a dict like `{'ds': Types.String, 'routing_group': Types.String}`

Reply:
though i do see how it is nice to use it in the constructor above. so i'll leave the ultimate determination up to you.